docs/events/glides/com.mbta.ctd.glides.operator_signed_in.v1.mdx

```diff
@@ -36,6 +36,7 @@ How the operator signed that they were fit for duty.
 Fields:
 
 - `type` (string): How the the operator made their signature.
+- `version` (integer): The version of the sign-in text the operator agreed to.
```

Comment:
question: Is the expectation that version is now a required field? If so, this would require a new event, `com.mbta.ctd.glides.operator_signed_in.v2`.

Reply:
I think this should be an optional field. It would mean the change is backwards compatible so we don't need to bump the schema version, and the event is still very meaningful without the text version.